### PR TITLE
Add German Translation

### PR DESCRIPTION
**Pull request by 6543**
and a smal fix

### CODE REVIEW

File: web/src/assets/locales/de.json [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        },
+        "events": {
+          "events": "Verfügbar bei folgenden Ereignissen",
+          "pr_warning": "Sei bitte vorsichtig mit dieser Option, da ein bösartiger Akteur eine bösartigen Pull Request erstellen kann, der deine Geheimnisse preisgibt."
+        }
+      },
+
+      "registries": {
+        "registries": "Registries",
+        "creds": "Zugangsdaten für die Registry",
```
[the remark this answers is not in the document]
Comment (6543) [reply web/src/assets/locales/de.json:126]: ich tendiere zu die ... war in dem was ich so umgangssprachlich gehört habe auch am heufigsten vertreten

File: web/src/assets/locales/de.json [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        "desc": "Zugangsdaten für die Registry können hinzugefügt werden, um private Images für Ihre Pipeline zu verwenden.",
+        "show": "Registries anzeigen",
+        "add": "Registry hinzufügen",
+        "none": "Es gibt noch keine Zugangsdaten für die Registry.",
+        "save": "Registry speichern",
+        "created": "Registry Zugangsdaten erstellt",
+        "saved": "Registry Zugangsdaten gespeichert",
+        "deleted": "Registry Zugangsdaten gelöscht",
```
Comment (6543) [inline web/src/assets/locales/de.json:134]:
```suggestion
        "created": "Registry-Zugangsdaten erstellt",
        "saved": "Registry-Zugangsdaten gespeichert",
        "deleted": "Registry-Zugangsdaten gelöscht",
```

File: web/src/assets/locales/de.json [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        },
+
+        "cancel_prev": {
+          "cancel": "Breche ältere Pipelines ab",
+          "desc": "Aktiviere diese Option, um laufende Pipelines desselben Ereignisses und Kontexts abzubrechen, bevor die neue Pipeline startet."
+        }
+      },
+
+      "secrets": {
+        "secrets": "Geheimnisse",
```
Comment (anbraten) [inline web/src/assets/locales/de.json:102]: I am not sure if we should call it `Geheimnisse`. Secrets is a pretty well known term in this topic IMO and calling secrets Geheimnisse feels to me like calling pipeline `Rohrleitung` 😉

File: web/src/assets/locales/de.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (anbraten) [reply web/src/assets/locales/de.json:126]: I would call it `die` as well.

File: web/src/assets/locales/de.json [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+        "add": "Registry hinzufügen",
+        "none": "Es gibt noch keine Zugangsdaten für die Registry.",
+        "save": "Registry speichern",
+        "created": "Registry-Zugangsdaten erstellt",
+        "saved": "Registry-Zugangsdaten gespeichert",
+        "deleted": "Registry-Zugangsdaten gelöscht",
+
+        "address": {
+          "address": "Adresse",
+          "placeholder": "Registratur Adresse (z. B. docker.io)"
```
[the remark this answers is not in the document]
Comment (anbraten) [reply web/src/assets/locales/de.json:138]: To be consistent:

```suggestion
          "placeholder": "Registry Adresse (z. B. docker.io)"
```

File: web/src/assets/locales/de.json [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+          "repair": "Repository reparieren",
+          "success": "Repository repariert"
+        },
+        "disable": {
+          "disable": "Repository deaktivieren",
+          "success": "Repository deaktiviert"
+        },
+        "delete": {
+          "delete": "Repository löschen",
+          "confirm": "Alle Daten sind nach dieser Aktion verloren!!!\n\nMöchten Sie wirklich fortfahren?",
```
Comment (anbraten) [inline web/src/assets/locales/de.json:161]: Looks like we used the du way in most other translations (which I would prefer over using Sie in general as well).

```suggestion
          "confirm": "Alle Daten sind nach dieser Aktion verloren!!!\n\nMöchtest du wirklich fortfahren?",
```

File: web/src/assets/locales/de.json [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+  },
+
+  "user": {
+    "oauth_error": "Fehler bei der Authentifizierung gegen OAuth-Anbieter",
+    "internal_error": "Ein interner Fehler ist aufgetreten",
+    "access_denied": "Du bist nicht berechtigt, dich anzumelden",
+    "token": "Dein persönlicher Token",
+    "shell_setup": "Kommando-Zeilen-Einrichtung",
+    "api_usage": "Beispiel für die API-Nutzung",
+    "cli_usage": "Beispiel für die Kommando-Zeilen-Nutzung",
```
Comment (anbraten) [inline web/src/assets/locales/de.json:213]:
```suggestion
    "cli_usage": "Beispiel für die Kommandozeilen-Nutzung",
```

File: web/src/assets/locales/de.json [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+      }
+    }
+  },
+
+  "user": {
+    "oauth_error": "Fehler bei der Authentifizierung gegen OAuth-Anbieter",
+    "internal_error": "Ein interner Fehler ist aufgetreten",
+    "access_denied": "Du bist nicht berechtigt, dich anzumelden",
+    "token": "Dein persönlicher Token",
+    "shell_setup": "Kommando-Zeilen-Einrichtung",
```
[the remark this answers is not in the document]
Comment (anbraten) [reply web/src/assets/locales/de.json:211]:
```suggestion
    "shell_setup": "Kommandozeilen-Einrichtung",
```

File: web/src/assets/locales/de.json [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+    "api_usage": "Beispiel für die API-Nutzung",
+    "cli_usage": "Beispiel für die Kommando-Zeilen-Nutzung",
+    "dl_cli": "Download Kommando-Zeilen-Tool",
+    "shell_setup_before": "Führen Sie die Schritte zur Einrichtung der Kommando-Zeile vorher aus"
```
Comment (anbraten) [inline web/src/assets/locales/de.json:215]:
```suggestion
    "shell_setup_before": "Führe bitte die Schritte zur Einrichtung der Kommandozeile vorher aus"
```

File: web/src/assets/locales/de.json [hunk 9]
```diff
@@ -0,0 +1,217 @@
...
+        "visibility": {
+          "visibility": "Sichtbarkeit des Projekts",
+
+          "public": {
+            "public": "Öffentlich",
+            "desc": "Jeder Benutzer kann Ihr Projekt sehen, ohne eingeloggt zu sein."
+          },
+          "private": {
+            "private": "Privat",
+            "desc": "Nur Sie und andere Besitzer des Repositorys können dieses Projekt sehen."
```
Comment (anbraten) [inline web/src/assets/locales/de.json:82]:
```suggestion
            "desc": "Nur du und andere Besitzer des Repositorys können dieses Projekt sehen."
```

File: web/src/assets/locales/de.json (hunk 3 above)
Comment (6543) [reply web/src/assets/locales/de.json:102]: oh das hätte was :laughing:

File: web/src/assets/locales/de.json [hunk 10]
```diff
@@ -0,0 +1,217 @@
...
+
+        "images": {
+          "images": "Verfügbar für folgende Images",
+          "desc": "Durch Kommata getrennte Liste der Images, für die dieses Geheimnis verfügbar ist; leer lassen, um alle Images zuzulassen"
+        },
+        "events": {
+          "events": "Verfügbar bei folgenden Ereignissen",
+          "pr_warning": "Sei bitte vorsichtig mit dieser Option, da ein bösartiger Akteur eine bösartigen Pull Request erstellen kann, der deine Geheimnisse preisgibt."
+        }
+      },
```
Comment (6543) [inline web/src/assets/locales/de.json:122]:
```suggestion
        "secrets": "Secrets",
        "desc": "Secrets können zur Laufzeit als Umgebungsvariablen an einzelne Pipelineschritte übergeben werden.",
        "none": "Es gibt noch keine Secrets.",
        "add": "Secret hinzufügen",
        "save": "Secret speichern",
        "show": "Secret anzeigen",
        "name": "Name",
        "value": "Wert",
        "deleted": "Secret gelöscht",
        "created": "Secret erstellt",
        "saved": "Secret gespeichert",

        "images": {
          "images": "Verfügbar für folgende Images",
          "desc": "Durch Kommata getrennte Liste der Images, für die dieses Secret verfügbar ist; leer lassen, um alle Images zuzulassen"
        },
        "events": {
          "events": "Verfügbar bei folgenden Ereignissen",
          "pr_warning": "Sei bitte vorsichtig mit dieser Option, da ein bösartiger Akteur eine bösartigen Pull Request erstellen kann, der deine Secrets preisgibt."
        }
      },
```

**Review by anbraten — APPROVED** [review]

**Pull request merged**